assets/js/modules/analytics/constants.js

```diff
@@ -17,3 +17,6 @@
  */
 
 export const AREA_MODULE_ANALYTICS_MAIN = 'moduleAnalyticsMain';
+
+// Date that Universal Analytics will stop recording data: July 1, 2023.
+export const UA_CUTOFF_DATE = '2023-07-01';
```

Comment:
It felt more logical to use the headline date that UA stops recording data on for the `UA_CUTOFF_DATE` constant, rather than June 30, 2023 as specified in the IB. So I have used that date and tweaked the date conditions accordingly.